src/style.css

```diff
@@ -0,0 +1,79 @@
+@font-face {
+  font-family: "Avenir";
+  src: url(fonts/Avenir-Book.ttf);
+}
+
+@font-face {
+  font-family: "Avenir";
+  font-weight: 800;
+  src: url(fonts/Avenir-Heavy.ttf);
+}
+
+html {
+  font-family: "Avenir", Arial, Helvetica, sans-serif;
+}
+
+.image {
+  position: absolute;
+  display: inline-block;
+}
+
+.imageBig {
+  margin-top: 25px;
+  margin-left: 26px;
+  margin-bottom: 26px;
+  width: 19px;
+  height: 19px;
+}
+
+.imageSmall {
+  margin-top: 14px;
+  margin-left: 12px;
+  margin-bottom: 15px;
+  width: 11px;
+  height: 11px;
+}
+
+.search {
+  position: relative;
+  display: flex;
+  margin-top: 20px;
+  background: rgb(255, 255, 255);
+  border: 1px solid rgb(217, 224, 232);
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 3px;
+
+}
+
+.searchText {
+  border: none;
+  outline: none;
+  width: 100%;
+  background-color: transparent;
+  font-family: inherit;
+}
+
+.searchTextSmall {
+  font-size: 14px;
+  margin-left: 33px;
+  height: 38px;
+}
+
+.searchTextBig {
+  font-size: 16px;
+  margin-left: 62px;
+  height: 66px;
+}
+
+.search:hover {
+  background: rgb(	255, 255, 255);
+  border: 1px solid rgb(217, 224, 232);
+  box-shadow: 0 3px 8px rgba(61, 78, 97, 0.2);
+}
+
+.search:focus-within {
+  background: linear-gradient(
+    180deg, hsl(0, 0%, 100%) 0%, hsl(240, 6%, 97%) 100%
+  );
+  border: 1px solid hsl(210, 25%, 91%);
```

Comment:
use or hex or rgb(-a) 

<img width="436" alt="image" src="https://user-images.githubusercontent.com/86183007/185241296-09391615-0728-476c-a912-a4991f93bb92.png">


```suggestion
  border-color: hsl(210, 25%, 91%);
```